apps/emqx/include/emqx.hrl

```diff
@@ -103,7 +103,7 @@
     by :: binary(),
     reason :: binary(),
     at :: integer(),
-    until :: integer()
+    until :: integer() | infinity
```

Comment:
Nit: this is mostly equivalent to the builtin `timeout()` type: https://www.erlang.org/doc/apps/erts/erlang#t:timeout/0